src/joystick/linux/SDL_sysjoystick.c

```diff
@@ -1449,6 +1449,7 @@ static void HandleInputEvents(SDL_Joystick *joystick)
                         HandleHat(SDL_EVDEV_GetEventTimestamp(event), joystick, hat_index, code % 2, event->value);
                         break;
                     }
+                    SDL_FALLTHROUGH;
```

Comment:
Can you please double check the 2 fall throughs in this file?
Perhaps you want to ignore the hat instead of falling through.

Reply:
To me, it looks like the fall-through is intended -- same below in this same file. @slouken?

Reply:
Yes, these are intentional. If we aren't treating them as a hat, we're treating them as normal axes.